core/scrollbar.js

```diff
@@ -609,17 +609,21 @@ Blockly.Scrollbar.prototype.createDom_ = function(opt_class) {
   if (opt_class) {
     className += ' ' + opt_class;
   }
-  this.outerSvg_ = Blockly.utils.createSvgElement('svg', {'class': className},
-                                                  null);
+  this.outerSvg_ = Blockly.utils.createSvgElement(
+      'svg', {'class': className}, null);
   this.svgGroup_ = Blockly.utils.createSvgElement('g', {}, this.outerSvg_);
-  this.svgBackground_ = Blockly.utils.createSvgElement('rect',
-      {'class': 'blocklyScrollbarBackground'}, this.svgGroup_);
+  this.svgBackground_ = Blockly.utils.createSvgElement(
+      'rect', {'class': 'blocklyScrollbarBackground'}, this.svgGroup_);
   var radius = Math.floor((Blockly.Scrollbar.scrollbarThickness - 5) / 2);
-  this.svgHandle_ = Blockly.utils.createSvgElement('rect',
-      {'class': 'blocklyScrollbarHandle', 'rx': radius, 'ry': radius},
```

Comment:
Is a single line object literal no longer allowed?

Reply:
Still allowed.  Deciding between them is just personal preference/how readable it makes the line.  I liked it better this way.